iosHyperskillApp/iosHyperskillApp/Sources/Extensions/ThirdParty/MokoResources/ResourcesColorResourceThemed+UIColor.swift

```diff
@@ -1,8 +1,35 @@
 import shared
 import UIKit
 
-extension ResourcesColorResource.Themed {
+extension ColorDesc {
+    var uiColor: UIColor {
+        switch self {
+        case let descResource as ColorDescResource:
+            return descResource.resource.getUIColor()
+        case let singleColor as ColorDescSingle:
+            return singleColor.color.uiColor
+        case let themedColor as ColorDescThemed:
+            return themedColor.dynamicUIColor
+        default:
+            return UIColor(hex6: 0xffffff)
```

Comment:
@ivan-magda 
I decided to return the white color as a default value because all the colors are stored in the static variables and it's impossible to throw an error during static variable initialization

Reply:
May, there is a better solution

Reply:
> Maybe, there is a better solution

I need some time to check whats changed in moko resources

Reply:
@ivan-magda 
https://github.com/icerockdev/moko-resources/releases/tag/release%2F0.21.0

Reply:
There is a `getUIColor` extension for iOS
For [ColorResourse](https://github.com/icerockdev/moko-resources/blob/6b320c12e07e57c866d787841e8a5b41c583808d/resources/src/iosMain/kotlin/dev/icerock/moko/resources/ColorResourceExt.kt#L10)
And for [ColorDesc](https://github.com/icerockdev/moko-resources/blob/6b320c12e07e57c866d787841e8a5b41c583808d/resources/src/iosMain/kotlin/dev/icerock/moko/resources/desc/color/ColorDescExt.kt#L8)